deeds-dapp-webapp/src/main/webapp/vue-app/dapp/main-dapp.js

```diff
@@ -111,17 +111,8 @@ const vuetify = new Vuetify({
   },
 });
 
-function getQueryParam(paramName) {
-  if (!window.location.search?.length) {
-    return;
-  }
-  const uri = window.location.search.substring(1);
-  const params = new URLSearchParams(uri);
-  return params.get(paramName);
-}
-
 function getLanguage() {
-  const lang = getQueryParam('lang');
+  const lang = document.documentElement.lang;
   return lang || localStorage.getItem('deeds-selectedLanguage') || (navigator.language.indexOf('fr') === 0 ? 'fr' : 'en');
```

Comment:
One single source of truth

```suggestion
  return document.documentElement.lang || 'en';
```